pkg/terminal/command.go

```diff
@@ -356,6 +356,11 @@ If regex is specified only the functions matching it will be returned.`},
 	types [<regex>]
 
 If regex is specified only the types matching it will be returned.`},
+		{aliases: []string{"packages"}, cmdFn: packages, helpMsg: `Print list of packages.
```

Comment:
_scripts/gen-cli-docs.go needs to be rerun.

Reply:
Done.